src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -128,16 +147,46 @@ public XMLStructuredOutput getOutput() throws IOException {
         return new XMLStructuredOutput(outcall.postFromOutput());
     }
 
+    private InputStream getInputStream() throws IOException {
+        if (logger != null && logger.isFINE()) {
+            // log the request, even when parsing fails
+            try (InputStream body = outcall.getResponse().body()) {
```

Comment:
is this guaranteed to only execute once? (was we get the body twice...)

Reply:
Well if you call `#getInput()` multiple times it will probably fail, because the input stream was read before. So it does not matter much if this fails in `#getInput()` or here already, right?